views/partials/search-form-fields.html

```diff
@@ -183,7 +183,7 @@
   <div class="controls">
     <button type="submit"><span>Search</span></button>
     {% if advancedSearchIsVisible %}
```

Comment:
I can't see it unless I remove the condition in the template, weird... Is it showing properly on your setup?